plexapi/base.py

```diff
@@ -565,6 +565,14 @@ def isPartialObject(self):
         """ Returns True if this is not a full object. """
         return not self.isFullObject()
 
+    def isLocked(self, field_name: str):
+        """ Returns True if the specified field is locked, otherwise False.
+
+            Parameters:
+                field_name (str): The name of the field, represented as a string.
```

Comment:
Rename to `field`. 
```suggestion
                field (str): The name of the field.
```

Reply:
Changing the name here required other changes, so made those as well.